### PR TITLE
Update return type for Runnable.__or__

### PR DESCRIPTION
**Pull request by nfcampos**

### CODE REVIEW

File: libs/langchain/langchain/schema/runnable/base.py [hunk 1]
```diff
@@ -270,7 +270,7 @@ def __ror__(
             Callable[[Iterator[Other]], Iterator[Any]],
             Mapping[str, Union[Runnable[Other, Any], Callable[[Other], Any], Any]],
         ],
-    ) -> Runnable[Other, Output]:
+    ) -> RunnableSerializable[Other, Output]:
```
Comment (eyurtsev) [inline libs/langchain/langchain/schema/runnable/base.py:273]: Why is this necessary?

**Review by eyurtsev — APPROVED** [review]
not sure why type narrowing necessary here

File: libs/langchain/langchain/schema/runnable/base.py (hunk 1 above)
Comment (nfcampos) [reply libs/langchain/langchain/schema/runnable/base.py:273]: Because otherwise you don't get the `.configurable_fields` methods on the return value of a `|`
